script/hardhat/fork/govUpdateTest.ts

```diff
@@ -0,0 +1,326 @@
+  timeLockCritical = await Timelock.deploy(GOVERNOR_PROXY_MAINNET, TIMELOCK_DELAYS_MAINNET.CRITICAL);
+
+  const aclAddressPadded = await ethers.provider.getStorageAt(COMPTROLLER_PROXY_MAINNET, 40);
+  const aclAddress = await ethers.utils.defaultAbiCoder.decode(["address"], aclAddressPadded).toString();
+  accessControl = await ethers.getContractAt("IAccessControlManager", aclAddress);
+
+  // Give Permission to Timelocks
+  let tx = await accessControl
+    .connect(aclAdmin)
+    .giveCallPermission(COMPTROLLER_PROXY_MAINNET, "_setLiquidationIncentive(uint256)", timeLockFastTrack.address);
+  await tx.wait();
+  tx = await accessControl
+    .connect(aclAdmin)
+    .giveCallPermission(COMPTROLLER_PROXY_MAINNET, "_setLiquidationIncentive(uint256)", timeLockCritical.address);
+  await tx.wait();
+  governorProxy = await ethers.getContractAt("GovernorBravoDelegator", GOVERNOR_PROXY_MAINNET);
+
+  //Deploy new governance v4 impl
+  const GovernorBravo = await ethers.getContractFactory("GovernorBravoDelegate");
+  const governorBravoV4Impl = await GovernorBravo.deploy();
+  await governorProxy.connect(governorAdmin)._setImplementation(governorBravoV4Impl.address);
+
+  // Iniitalize impl via Proxy
+  governorProxy = await ethers.getContractAt("GovernorBravoDelegate", GOVERNOR_PROXY_MAINNET);
+  tx = await governorProxy
+    .connect(governorAdmin)
+    .initialize(
+      XVS_VAULT_MAINNET,
+      PROPOSAL_TYPE_CONFIGS,
+      [NORMAL_VIP_TIMELOCK, timeLockFastTrack.address, timeLockCritical.address],
+      await governorAdmin.getAddress(),
+    );
+  await tx.wait();
+};
+
+describe("Governance v4", () => {
+  const NEW_LIQ_INCENTIVE = parseUnits("2", 18);
+  describe("NORMAL VIP", () => {
+    const proposal = {
+      signatures: ["_setLiquidationIncentive(uint256)"],
+      targets: [COMPTROLLER_PROXY_MAINNET],
+      values: ["0"],
+      params: [[NEW_LIQ_INCENTIVE]],
+    };
+    let proposalId;
+    before(async () => {
+      await loadFixture(governanceFixture);
+    });
+    describe("Propose:", () => {
+      it("should revert if below votingThreshold", async () => {
+        const { targets, signatures, values } = proposal;
+        await expect(
+          governorProxy.propose(targets, values, signatures, getCalldatas(proposal), "VIP 1", PROPOSAL_TYPES.NORMAL),
+        ).to.be.revertedWith("GovernorBravo::propose: proposer votes below proposal threshold");
+      });
+      it("should pass", async () => {
+        const { targets, signatures, values } = proposal;
+        const proposalIdBefore = await governorProxy.callStatic.proposalCount();
+        const tx = await governorProxy
+          .connect(proposer)
+          .propose(targets, values, signatures, getCalldatas(proposal), "VIP 1", PROPOSAL_TYPES.NORMAL);
+        await tx.wait();
+        proposalId = await governorProxy.callStatic.proposalCount();
+        expect(proposalIdBefore.add(1)).to.equal(proposalId);
+      });
+    });
+    describe("Cast Vote:", () => {
```

Comment:
Can we add some spacing in between test suites and even inside test cases for readability? At minimum we should have a new line between suites and cases